docs/breakingchanges-0.63.md

```diff
@@ -0,0 +1,92 @@
+
+# Breaking changes for 0.63
+
+## Managed C# projects
+In 0.63 we have deprecated the Microsoft.ReactNative.SharedManaged project in favor of a proper library reference. If you only have cpp projects, no action is needed. If you have a C# app, to update your project please perform the following actions:
+
+### At least one C# project
```

Comment:
>### [](start = 0, length = 3)

It looks too complex. Why not to instruct changing project references in Visual Studio?

Reply:
nit: c# app or native module

Reply:
@vmoroz: 2 reasons: a) I case folks don't have it installed b) it depends if they have the 0.62 updated template or not, there is no way to see this from VS.

Reply:
@dannyvv @vmoroz VS 2019 is a requirement for RNW, so I wouldn't worry about that.